.prettierignore

```diff
@@ -0,0 +1 @@
+examples/prettier/invalid.*
```

Comment:
disallow prettifying `examples/prettier/invalid.{js,graphql}` files, because they must be invalid